libraries/AC_AutoTune/AC_AutoTune_FreqResp.cpp

```diff
@@ -258,14 +262,14 @@ void AC_AutoTune_FreqResp::push_to_meas_buffer(uint16_t count, float amplitude,
     sample.curr_count = count;
     sample.amplitude = amplitude;
     sample.time_ms = time_ms;
-    meas_peak_info_buffer.push(sample);
+    meas_peak_info_buffer->push(sample);
 }
 
 // pull measured peak info from buffer
 void AC_AutoTune_FreqResp::pull_from_meas_buffer(uint16_t &count, float &amplitude, uint32_t &time_ms)
 {
     peak_info sample;
-    if (!meas_peak_info_buffer.pop(sample)) {
+    if (!meas_peak_info_buffer->pop(sample)) {
```

Comment:
```suggestion
    if ((meas_peak_info_buffer == nullptr) || !meas_peak_info_buffer->pop(sample)) {
```
Does this need nullptr check?

Reply:
done. THanks.